apiary.apib

```diff
@@ -824,6 +824,29 @@ Get a quotation
             + deal (object)
                 + type: `deal` (string)
                 + id: `53474a7a-f9b2-4dd4-88a8-40ce773c7a64` (string)
+            + grouped_lines (array)
+                + (object)
+                    + section (object)
+                        + title (string)
+                    + line_items (array)
+                        + (object)
+                            + product (object, nullable)
```

Comment:
Not sure if we should add `nullable` to read endpoints. But will do followup pr if we figure this out.

Reply:
I don't think `nullable` and `optional` are displayed in the documentation anyway?